tests/test_operation.py

```diff
@@ -38,12 +38,20 @@ def test_heisenberg(self):
         def h_test(cls):
             "Test a gaussian CV operation."
             log.debug('\tTesting: cls.{}'.format(cls.__name__))
+
+            ww = list(range(cls.num_wires))
+
             # fixed parameter values
             if cls.par_domain == 'A':
-                par = [nr.randn(1,1)] * cls.num_params
+                if cls.__name__ == "Interferometer":
+                    ww = list(range(2))
+                    par = [np.array([[0.83645892-0.40533293j, -0.20215326+0.30850569j],
+                                     [-0.23889780-0.28101519j, -0.88031770-0.29832709j]])]
+                else:
+                    par = [np.array([[-1.82624687]])] * cls.num_params
             else:
-                par = list(nr.randn(cls.num_params))
-            ww = list(range(cls.num_wires))
+                par = [-0.069125, 0.51778, 0.91133, 0.95904][:cls.num_params]
```

Comment:
ditto about magic numbers

Reply:
This should get fixed for all tests as per #99